src/copaw/agents/task_router.py

```
@@ -0,0 +1,382 @@
+    HEARTBEAT_PATTERNS = [
+        r"^ping$",
+        r"^pong$",
+        r"^status$",
+        r"^heartbeat$",
+        r"^health\s*check$",
+        r"^are you there",
+        r"^you there",
+        r"^你还在吗",
+        r"^在吗",
+        r"^还在吗",
+    ]
```

Ruff (0.15.2)

[warning] Mutable default value for class attribute

(RUF012)